### DISCUSSION TITLE
Manu Modus not persistent after RaspberryMatic update

### DISCUSSION THREAD

**Discussion by macbih** [OpenCCU/OpenCCU#2418] [🇺🇸 Q&A] [answered]
### Describe the issue you are experiencing

I operate Homematic IP devices locally on ELV Charly. The sensors and actuators are read out and controlled by a Home Assistant instance. Among other things, the air conditioning of the house depends on the correctly transmitted sensor values.

It has happened several times now after an update of RaspberryMatic (incl. restart of Home Assistant) that the actual temperatures of the thermostats (HmIP-eTRV-B / HmIP-STHD / HmIP-WTH-2) were reported to be 0° C. After checking in the RaspberryMatic WebUI, I found that the desired `Manu Modus` was no longer activated for the thermostats. Apparently, the thermostats were completely deactivated. I now had to manually set all my thermostats back to manual mode.

Side note:
As a workaround, I would like to set the manual mode automatically in Home Assistant, but unfortunately have not found a way to implement this.

### Describe the behavior you expected

The units concerned retain the set (Manu) mode even after updating RaspberryMatic.

### Steps to reproduce the issue

1. set thermostats to `Manu mode` within RaspberryMatic WebUI
2. watch correctly reported thermostat temperatures within Home Assistant
3. update RaspberryMatic 
4. watch thermostat temperatures within Home Assistant again
5. check whether `Manu mode` is still set within RaspberryMatic WebUI

### What is the version this bug report is based on?

3.71.12.20230826

### Which base platform are you running?

rpi3 (RaspberryPi3)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
n/a
```

### Additional information

![grafik](https://github.com/jens-maus/RaspberryMatic/assets/11271949/2cc4750a-fcfe-4e68-ab2a-2af6ff28916b)
After an update, all those 4 options incl. `Manu Modus` are deactivated.

**Comment by jens-maus** [comment, marked as answer]
> It has happened several times now after an update of RaspberryMatic (incl. restart of Home Assistant) that the actual temperatures of the thermostats (HmIP-eTRV-B / HmIP-STHD / HmIP-WTH-2) were reported to be 0° C.

Please note, that upon a restart of the CCU/RaspberryMatic it is totally __normal__ that battery-driven homematicIP sensors/actuators will first be shown with default values (e.g. 0° C) in the WebUI and that it might take a while (15 min till an hour) until these HmIP sensors/actuators are submitting their current values in a regular fashion. This is nothing RaspberryMatic or a CCU can modify as this is a normal behaviour of certain battery-driven HomeMatic/homematicIP devices - simply to safe energy and RF transmission time (DutyCycle).

> After checking in the RaspberryMatic WebUI, I found that the desired `Manu Modus` was no longer activated for the thermostats. Apparently, the thermostats were completely deactivated. I now had to manually set all my thermostats back to manual mode.

Please, next time check the actual status of these thermostats on-site and not just in the WebUI. As said, after a fresh reboot of a CCU/RaspberryMatic it might take a while until all homematicIP devices have retransmitted their current values, including Manu/Auto-Modus status, etc.). But the devices themselves should continue to work in the last set mode – just the shown values/status in the WebUI, thus in the supplied interfaces and thus in HomeAssistant might not be in sync. That's exactly the reason why a CCU/RaspberryMatic should **NOT** be reguarly rebooted and that you should give it some time to actually "sync" all values of all homematic devices over a certain time.

FYI: As this seems to be a pure user problem and not a common issue or bug, I moved it to the discussion part.

**Comment by macbih** [comment]
Thank you very much, @jens-maus, for your helpful explanation! That makes much sense to me and now I know how to handle this.
